src/design-system/components/switch/Switch.tsx

```diff
@@ -0,0 +1,33 @@
+import { SizeEnum } from '~/types'
+import Switch, { SwitchProps } from '@mui/material/Switch'
+import { FormControlLabel } from '@mui/material'
+import './Switch.scss'
+interface AppSwitchProps extends Omit<SwitchProps, 'size'> {
+  labelPosition?: 'start' | 'end' | 'top' | 'bottom'
+  size?: SizeEnum
+  label?: string
+  loading?: boolean
+}
+export const AppSwitch = ({
+  labelPosition = 'end',
+  size = SizeEnum.Medium,
+  label,
+  loading,
+  disabled,
+  ...props
+}: AppSwitchProps) => {
+  return (
+    <FormControlLabel
+      className={`s2s-form-label-box s2s-form-label-box--${size}`}
+      control={
+        <Switch
+          className={`s2s-switch--${size}`}
+          disabled={loading || disabled}
+          {...props}
+        />
+      }
+      label={label ?? ''}
```

Comment:
```suggestion
      label={label}
```
Maybe add `' '` to default value of label. I think it would be better here, as all default values defined there.